utils/m0reportbug

```diff
@@ -886,6 +894,20 @@ optparse() {
             --addb) COLLECT_ADDB_P=1;;
             --skip-addb-files) SKIP_ADDB_FILE_P=1;;
             -b|--binaries) COLLECT_BINARIES_P=1;;
+            --coredumps) 
+                if [ -n "$2" ];
```

Comment:
As discussed, get rid off this condition and validate --coredump option to be either "true" or "false".

Reply:
incorporated